packages/devtools_app/lib/src/analytics/constants.dart

```diff
@@ -95,24 +94,6 @@ const profileAppStartUp = 'profileAppStartUp';
 const cpuProfileFlameChartHelp = 'cpuProfileFlameChartHelp';
 const cpuProfileProcessingTime = 'cpuProfileProcessingTime';
 
-// Memory UX actions:
-const gc = 'gc';
-const memoryLegend = 'memoryLegend';
-const memorySettings = 'memorySettings';
-const androidChart = 'androidChart';
-const autoSnapshot = 'autoSnapshot';
-const showChart = 'showChart';
-const hideChart = 'hideChart';
```

Comment:
We shouldn't change the strings here (I see we are now using 'chartExpand' and 'chartCollapse'). Changing the strings will prevent us from being able to compare current data with historical data in our analytics

Reply:
we are still changing the analytics for hiding and showing the chart, which we should not do for existing analytics. Can we switch back to using showChart and hideChart?